express-notes-api/index.js

```diff
@@ -0,0 +1,122 @@
+
+const express = require('express');
+const fs = require('fs');
+const data = require('./data.json');
+
+const app = express();
+const jsonMiddleware = express.json();
+
+const noteArray = [];
+const dataNotes = data.notes;
+
+if (Object.keys(dataNotes).length !== 0) {
+
+  for (const id in dataNotes) {
+    noteArray.push(dataNotes[id]);
+  }
+}
```

Comment:
I recommend only creating this array on-demand, inside your `GET /api/notes` callback.